server/controllers/startPage/startPage.ts

```diff
@@ -1,12 +1,25 @@
 import { Request, Response } from 'express'
+import jwtDecode from 'jwt-decode'
 import config from '../../config'
 import { isBannerDisplayDateRangeValid } from '../../utils/utils'
 
+export enum HMPPS_AUTH_ROLE {
+  PPCS = 'ROLE_MARD_PPCS',
+}
```

Comment:
The existing roles we have are:
ROLE_MAKE_RECALL_DECISION
ROLE_MAKE_RECALL_DECISION_SPO

It would be much better if this new one conformed to the same pattern. i.e.
ROLE_MAKE_RECALL_DECISION_PPCS

Reply:
Eek! The HMPPS-Auth team, and Sam have already added that name to dev and to the hmpps-auth service for local use!
Er... shall I create new PRs/requests to change that? Or are we ok to keep it as it is?

Reply:
I think it would be better to change it.  A little bit of pain now will avoid possibly frequent confusion/frustration/mistakes in the future.

Reply:
Sounds good!

Reply:
Done 👍